hbase-protocol-shaded/src/main/protobuf/Snapshot.proto

```diff
@@ -35,15 +35,16 @@ message SnapshotDescription {
   required string name = 1;
   optional string table = 2; // not needed for delete, but checked for in taking snapshot
   optional int64 creation_time = 3 [default = 0];
+  optional int64 ttl = 4 [default = 0];
```

Comment:
It will break wire compatibility, ttl should be appended at the end of parameters sequence.

Reply:
Makes sense. Will change this